test/helpers/create-https-test-server.ts

```diff
@@ -68,7 +69,7 @@ const createHttpsTestServer = async (options: HttpsServerOptions = {}): Promise<
 
 	await pify(server.https.listen.bind(server.https))();
 
-	server.caKey = caKey;
+	server.caKey = caKey as any;
```

Comment:
I wasn't sure what the exact meaning of this code, so I just added `as any` just in case if this assignment was without type casting to Buffer by intention. Should I convert it to buffer?